modules/reference/pages/rpk/rpk-cluster/rpk-cluster-self-test-start.adoc

```diff
@@ -14,16 +16,15 @@ Available tests to run:
 ** Throughput test: 8192-bit messages
 *** Unique pairs of Redpanda nodes each act as a client and a server.
 *** The test pushes as much data over the wire, within the test parameters.
-* *Cloud tests*
+* *Cloud storage tests*
 ** Latency test: 1024-bit object.
 ** Depending on cluster read/write permissions (xref:reference:properties/object-storage-properties.adoc#cloud_storage_enable_remote_read[`cloud_storage_enable_remote_read`], xref:reference:properties/object-storage-properties.adoc#cloud_storage_enable_remote_write[`cloud_storage_enable_remote_write`]), a series of cloud storage operations are performed:
 *** Upload an object to an object storage.
 *** List objects in the object storage.
 *** Download an object from the object storage.
 *** Delete the original object from the object storage, if it was uploaded.
 
-This command immediately returns on success, and the tests run asynchronously. The
-user polls for results with the xref:./rpk-cluster-self-test-status.adoc[rpk cluster self-test status]
+This command asks for users confirmation (unless when the flag `--no-confirm` is used.), then returns with a test identifier ID, and starts to run the tests run. To see the status and the results pool with the xref:./rpk-cluster-self-test-status.adoc[rpk cluster self-test status]
```

Comment:
```suggestion
This command prompts users for confirmation (unless the flag `--no-confirm` is specified), then returns a test identifier ID, and runs the tests. To view the status and the results pool, run xref:./rpk-cluster-self-test-status.adoc[rpk cluster self-test status].
```

Reply:
```suggestion
This command prompts users for confirmation (unless the flag `--no-confirm` is specified), then returns a test identifier ID, and runs the tests. To view the test status, run a pool with xref:./rpk-cluster-self-test-status.adoc[rpk cluster self-test status]. Once the tests end, see the cached results with the same status command.
```

Reply:
Rewrote to better convey the actual implementation. What happens is that you run the test with `rpk cluster self-test start` then the command asks for confirmation. After confirming, it starts and the command stops since the tests run async. To check the status, we have to [poll](https://en.wikipedia.org/wiki/Polling_(computer_science)#:~:text=Polling%20is%20the%20process%20where,often%20with%20low%2Dlevel%20hardware.) the status with `rpk cluster self-test status` . 

When the tests finish, the cached results will be displayed on the same command.